docs/source/sci_api_tutorial/add_missing_residues.rst

```diff
@@ -0,0 +1,138 @@
+============================================
+Adding Missing Residues
+============================================
+
+
+
+Briefs
+============================================
+
+Proteins are often missing residues from flexible loop regions. This is a problem for various types
+of calculations including molecular dynamics (MD) and quantum mechanics (QM) simulations. This function
+aids in HTP workflow design by adding missing ``Residue()`` objects into a ``Structure()``. Missing residues
+can be defined by the user or by pulling from the Protein DataBank (PDB) when possible.
+
+.. note:: 
+    
+    **Using this functionality requires the modeller python package to be installed.**
+    For more information on installing the package, see the `Sali Lab website <https://salilab.org/modeller/>`_
+
+
+.. note::
+
+    Science API is a special concept in EnzyHTP. They stands for those top-layer APIs
+    the are supposed to be used in assembling the `workflow <https://enzyhtp-doc.readthedocs.io/en/latest/sci_api_tutorial/how_to_assemble.html#find-the-science-api-that-directly-gives-what-you-need>`_.
+
+.. warning::
+    
+    Some loop refinement methods are known to slightly move sidechain positions. It may be necessary to minimize a ``Structure`` after filling in missing loops.
+
+**You will find a detailed tutorial of how to use this Science API.**
+
+Input/Output
+=========================================================================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/add_missing_residues.svg
```

Comment:
Please adjust the layout to make font larger

Reply:
Added a version with a larger text size